internal/appliance.go

```diff
@@ -0,0 +1,15 @@
+package internal
+
+type Appliance struct {
+	Name string
+	Consumption int
+	Duration int
```

Comment:
Si no habéis planteado el problema, es imposible que pongáis una solución correcta. ¿Qué es el consumo? ¿Kilowatios? ¿Watios? ¿Es fijo o depende de algo? Si es consumo medio, ¿será un número real=

Igual la duración. ¿Son horas? ¿Minutos? ¿Segundos?. Insisto, debéis plantear los issues como problemas. Si no, simplemente no estáis haciendo nada.

Reply:
Nos ponemos a ello.